libfreerdp/core/gateway/rdg.c

```diff
@@ -104,6 +104,10 @@
 #define HTTP_CAPABILITY_REAUTH 0x10
 #define HTTP_CAPABILITY_UDP_TRANSPORT 0x20
 
+/* http response protocol encoding */
+#define HTTP_TRANSFER_ENCODING_IDENTITY 0x01
```

Comment:
I know we used to have lots of defines, but we tend to use enums when possible now in new code.

Reply:
of course doable, but since above everything was a define I continued that way...